### ISSUE TITLE
Invitation email does not include link to portal

### ISSUE THREAD

**Issue by marwenmema** [exo-addons/external-user-invitation#1]
@mgreau This would be a very useful add-on and should even become a standard feature IMO (cc @plamarque ). I'm using it for a PoC. _However_, the invite provides the invitee with a login and password but no link to the portal, rendering the add-on useless (at least for the use cases that it's usually being needed for).

**Comment by boubaker** [comment]
In fact you could edit portal URL in the mails templates WCM contents, see section "Mail Templates" in this documentation:
https://community.exoplatform.com/portal/g/:spaces:external_users_addin/external_users_addin/wiki/Administration_documentation

**Comment by marwenmema** [comment]
Awesome. Thanks @boubaker 
I added this information to the README.md

**Comment by plamarque** [comment]
@marwenmema  I agree the collaboration with external people is an axis we need to tackle seriously. 
You will show me how this works cause I didn't find any screenshot in the docs.

@hghorbel please check if we have some related PFRs already for this

**Comment by boubaker** [comment]
> we need to tackle seriously

I think that this addon is serious too. That's why it's used by two of our customers ;-)

**Comment by boubaker** [comment]
@marwenmema , please don't commit directly on Addon and do a Pull Request so that I can review and validate, even on a README file. This strategy is used for all parts of our product.

**Comment by boubaker** [comment]
@marwenmema , I reverted the documentation part that was added, since I would like to add a more complete and structured documentation as done in linked Wiki page (Keep just the links or copy everything)
